docs/src/main/paradox/release-notes/10.1.x.md

```diff
@@ -1,5 +1,43 @@
 # 10.1.x Release Notes
 
+## 10.1.15
+
+This release is an important security fix release, backporting fixes from 10.2.7 to the 10.1.x line of releases. Note,
+that we do not plan any further releases for 10.1.x, so please consider upgrading soon to the latest supported release line,
+10.2.x.
```

Comment:
Maybe add something along the lines of: "Due to our binary compatibility guarantees, as long as no deprecated methods were used 10.2.x should be a drop-in replacement in 10.1.x projects, even when transitive dependencies depended on 10.1.x."?

Reply:
```suggestion
10.2.x. Due to our binary compatibility guarantees, as long as no deprecated methods were used, 10.2.x should be a drop-in replacement in 10.1.x projects, even when transitive dependencies depended on 10.1.x.
```